app/code/core/Mage/Adminhtml/Block/Catalog/Category/Abstract.php

```diff
@@ -93,8 +93,9 @@ public function getRoot($parentNodeCategory = null, $recursionLevel = 3)
             $tree = Mage::getResourceSingleton('catalog/category_tree')
                 ->load(null, $recursionLevel);
 
-            if ($this->getCategory()) {
-                $tree->loadEnsuredNodes($this->getCategory(), $tree->getNodeById($rootId));
+            $category = $this->getCategory();
+            if (is_object($category) and $category->getId()) {
```

Comment:
Not a big deal, but using `&&` would be more consistent with the codebase.

```php
if (is_object($category) && $category->getId()) {
```

Reply:
true, done

Reply:
what is this change for?